packages/notification-center/src/utils/react-to-webcomponent.ts

```diff
@@ -0,0 +1,263 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
The `react-to-webcomponent` library code, which is adjusted to our needs.
It creates the proxy component around the react component, and it mounts it directly to the web component container.